beacon-chain/powchain/service.go

```diff
@@ -50,27 +51,27 @@ type Client interface {
 // Validator Registration Contract on the PoW chain to kick off the beacon
```

Comment:
Deposit Contract

Reply:
Did we rename it? This would be a renaming in another PR, in my opinion

Reply:
Yeah the term VRC/ValidatorRegistrationContract is used in a lot of places. I can perform a sweep across the repo and change it in another PR